recipes/torchbiggraph/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "torchbiggraph" %}
+{% set version = "1.dev1" %}
```

Comment:
is there a non-development version we can use?

Reply:
Not yet. There have only been two releases, both as development versions

Reply:
How long will it be until a full release? We typically don't package development releases

Reply:
I'm not 100%. I've reached out the the source maintainer

Reply:
@scopatz https://github.com/facebookresearch/PyTorch-BigGraph/issues/84#issuecomment-511928482